content/en/news/releases/1.16.x/announcing-1.16/change-notes/index.md

```diff
@@ -0,0 +1,141 @@
+  if `ENABLE_ENHANCED_RESOURCE_SCOPING` feature flag is enabled.
+
+- **Fixed** `LocalityLoadBalancerSetting.failoverPriority` not working properly if xDS cache is enabled.
+  ([Issue #40198](https://github.com/istio/istio/issues/40198))
+
+- **Fixed** disable `PILOT_ENABLE_CONFIG_DISTRIBUTION_TRACKING` temporarily to fix some memory/CPU cost issues.
+
+- **Fixed** an issue where Remote JWKS URI's without a host port fail to parse into their host and port components.
+
+- **Fixed** the ordering of RBAC and metadata exchange filters while generating HTTP/network filters
+  ([Issue #41066](https://github.com/istio/istio/issues/41066))
+
+- **Fixed** an issue causing traffic to not match (and return a `404`) when using wildcard domain names and including an unexpected port in the `Host` header.
+
+- **Fixed** an issue causing traffic to match an unexpected route when using wildcard domain names and including an port in the `Host` header.
+
+## Security
+
+- **Improved** Pilot will now load its DNS serving certificate from well known locations:
+
+{{< text plain >}}
+/var/run/secrets/istiod/tls/tls.crt
+/var/run/secrets/istiod/tls/tls.key
+/var/run/secrets/istiod/ca/root-cert.pem
+{{< /text >}}
+
+The CA path will alternatively be loaded from: `/var/run/secrets/tls/ca.crt`
+It also automatically loads any secret called `istiod-tls` and the `istio-root-ca-configmap` into those paths.
+This method is preferred to use those well known paths than to set the TLS arguments.
+This will allow for an easier installation process for `istio-csr` as well as any other external issuer that needs to modify
+the Pilot DNS serving certificate. ([Issue #36916](https://github.com/istio/istio/issues/36916))
+
+- **Updated** dependency in Envoy to properly parse JWTs with negative values for exp, nbf or iat fields.
+
+## Telemetry
+
+- **Updated** Telemetry API uses a new native extension for Prometheus stats
+  instead of the Wasm-based extension. This improves CPU overhead and memory
+  usage of the feature. Custom dimensions no longer require regex and bootstrap
+  annotations. If customizations use CEL expressions with Wasm attributes, they
+  are likely to be affected. This change can be disabled by setting the control
+  plane feature flag `TELEMETRY_USE_NATIVE_STATS` to `false`.
+
+- **Added** support for use of the OpenTelemetry tracing provider with the Telemetry API.
+  ([Issue #40027](https://github.com/istio/istio/issues/40027))
+
+- **Fixed** an issue to allow multiple regular expressions with the same tag name.
+  ([Issue #39903](https://github.com/istio/istio/issues/39903))
+
+## Extensibility
+
+- **Improved** When Wasm module downloading fails and `fail_open` is true, a RBAC filter allowing all the traffics is passed to Envoy instead of the original Wasm filter.
+  Previously, the given Wasm filter itself was passed to Envoy in the case, but it may cause the errors because some fields of Wasm configuration are optional in Istio, but not in Envoy.
```

Comment:
```suggestion
  Previously, the given Wasm filter itself was passed to Envoy in this case, but it may cause the errors because some fields of Wasm configuration are optional in Istio, but not in Envoy.
```